README.md

```diff
@@ -17,6 +17,9 @@ details.
 - JupyterLab 4.x (for newer Jupyter Scheduler versions)
 - JupyterLab 3.x (for Jupyter Scheduler 1.x)
 
+> [!IMPORTANT]
+> Starting with v2.7.0, Jupyter Scheduler requires SQLAlchemy 2.x instead of SQLAlchemy 1.x.
+
```

Comment:
I know this line isn't modified as part of this PR, but line 24 can now start, "JupyterLab 3 reached its end of maintenance …".

Reply:
Changed "will reach" to "reached" in both README and readthedocs.